docs/data/routes/docs/nextjs/internationalization/using-i18n/en.md

```diff
@@ -3,4 +3,77 @@ name: using-i18n
 routeTemplate: ./data/component-templates/article.yml
 title: Using Next.js i18n
 ---
-# Using Next.js i18n with JSS
\ No newline at end of file
+# Using Next.js i18n with JSS
+
+Next.js provides a way to use [Internationalized Routing](https://nextjs.org/docs/advanced-features/i18n-routing).
+The sample app is using [Sub-path Routing](https://nextjs.org/docs/advanced-features/i18n-routing#sub-path-routing).
```

Comment:
We should perhaps emphasize that domain routing would require customization to implement.

Reply:
I added a note on that. See 30b2301c9fe17314c398aa09ba8f47b883844817. Let me know if we need to elaborate.